app/main/helpers/framework_helpers.py

```diff
@@ -0,0 +1,9 @@
+def get_latest_live_framework(all_frameworks, framework_type):
+    try:
+        latest = max(
+            (f for f in all_frameworks if f['status'] == 'live' and f['framework'] == framework_type),
+            key=lambda f: f['id'],
```

Comment:
Using ID for this really stinks.  (Yes, I know that's how it was doing it before, and no, I don't have a better suggestion right now).